MH4U Dex/ItemEncyclopediaTableViewController.m

```diff
@@ -46,12 +42,12 @@ - (void)viewDidLoad
 
 - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 {
-    return 1;
+    return self.fetchedResultsController.sections.count;
 }
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    return self.items.count;
+    return [[self.fetchedResultsController.sections objectAtIndex:section] numberOfObjects];
```

Comment:
Since `sections` is an array, you could do this if you want to:
```objc
return [self.fetchedResultsController.sections[section] numberOfObjects];
```